common/src/main/java/com/ultreon/devices/core/Laptop.java

```diff
@@ -451,16 +451,16 @@ public void renderLaptop(final @NotNull PoseStack pose, final int mouseX, final
         //************************************//
         // Draw the watermark on the desktop. //
         //************************************//
-        if (!Devices.DEVELOPER_MODE) {
-            if (Devices.isDevelopmentPreview()) {
-                drawString(pose, getFont(), "Development Preview - " + Reference.VERSION, posX + BORDER + 5, posY + BORDER + 5, Color.WHITE.getRGB());
-            } else {
-                drawString(pose, getFont(), "Alpha " + Reference.VERSION, posX + BORDER + 5, posY + BORDER + 5, Color.WHITE.getRGB());
-            }
-        } else {
-            drawString(pose, getFont(), "Developer Version - " + Reference.VERSION, posX + BORDER + 5, posY + BORDER + 5, Color.WHITE.getRGB());
-        }
-
+        //if (!Devices.DEVELOPER_MODE) {
+        //     if (Devices.isDevelopmentPreview()) {
+        //         drawString(pose, getFont(), "Development Preview - " + Reference.VERSION, posX + BORDER + 5, posY + BORDER + 5, Color.WHITE.getRGB());
+        //     } else {
+        //         drawString(pose, getFont(), "Alpha " + Reference.VERSION, posX + BORDER + 5, posY + BORDER + 5, Color.WHITE.getRGB());
+        //     }
+        // } else {
+        //     drawString(pose, getFont(), "Developer Version - " + Reference.VERSION, posX + BORDER + 5, posY + BORDER + 5, Color.WHITE.getRGB());
+        // }
+        drawString(pose, getFont(), "Beta " + Reference.VERSION, posX + BORDER + 5, posY + BORDER + 5, Color.WHITE.getRGB()); // TODO: figure out a better way to do this
```

Comment:
Rather allow `isDevelopmentPreview` to be used. As it would be used in exclusive previews that sometimes gets shared.

Reply:
> There's a TODO there, also was edited in a rush on https://vscode.dev

Where? @Jab125